pills/20/three-hellos.nix

```diff
@@ -36,6 +36,7 @@ let
       mkdir -p "$out/bin"
       echo "#! ${stdenv.shell}" >> "$out/bin/hello"
       echo "exec $(which hello)" >> "$out/bin/hello"
+      chmod 0744 "$out/bin/hello"
```

Comment:
I suggest using more natural `0755` permissions. Otherwise the change looks good.

Reply:
Done!